[INFO -- 2017-03-06 23:25:31]
Starting flip_value script
version: 2.0
[INFO -- 2017-03-06 23:25:31]
Initialization strings:
[INFO -- 2017-03-06 23:25:31]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-03-06 23:25:31]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-06 23:25:31]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-03-06 23:25:31]
(4) file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-06 23:25:31]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_check
[INFO -- 2017-03-06 23:25:31]
(6) set args 228 2048 16 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_a_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/dgemm_b_8192 /micNfs/gdb-fault-injection/injector/bin/dgemm/gold_2048_m-order_228_ths_16_blocks 1
[INFO -- 2017-03-06 23:25:31]
initSignal:14
[INFO -- 2017-03-06 23:25:31]
endSignal:21
[INFO -- 2017-03-06 23:25:31]
seqSignal:15
[INFO -- 2017-03-06 23:25:31]
faultMode:2
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[EXCEPTION -- 2017-03-06 23:26:03]
pythonException: invalid frame
[INFO -- 2017-03-06 23:26:03]
Backtrace BEGIN:
#0  0x000000000040369a in L_dgemm_185__par_region0_2_0 () at dgemm.c:220
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffe550 in ?? ()
#3  0x0000000000000000 in ?? ()

215	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
216	                            CC_arr(i,j) = 0.0;
217	
218	                    for (kg=kk,k=0; kg<MIN(kk+block,order); k++,kg++)
219	                        for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
220	                            for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)
221	                                CC_arr(i,j) += AA_arr(i,k)*BB_arr(j,k);
222	
223	                    for (jg=jj,j=0; jg<MIN(jj+block,order); j++,jg++)
224	                        for (ig=ii,i=0; ig<MIN(ii+block,order); i++,ig++)

Backtrace END
Memory content before bitflip:0x8d0xbb0x8d0xf70xff0x7f0x000x00
Fault Mode: Random bit-flip
base address to flip value: 0x7fffffffdf38
address max offset: 8
Memory content after  bitflip:0x330xa30x960x450x340x890x2a0x7b
frame name: L_dgemm_185__par_region0_2_0
symbol name: order
symbol filename: dgemm.c
symbol line: 181
value: 8875056873285788467
value address: 0x7fffffffdf38
Type: The type is an integer type.
Type sizeof: 8

[INFO -- 2017-03-06 23:26:03]
Fault Injection Successful after 32.2188899517s
[INFO -- 2017-03-06 23:26:28]
event type: exit
[INFO -- 2017-03-06 23:26:28]
exit code: 0
